Resources/doc/config.rst

```diff
@@ -405,9 +405,10 @@ Otherwise you will get a *auth failed* exception.
             connections:
                 default:
                     server: "mongodb://localhost:27017"
-                    username: someuser
-                    password: somepass
-                    authSource: db_you_have_access_to
+                    options:
+                        username: someuser
+                        password: somepass
+                        db: db_you_have_access_to
```

Comment:
Ah, so if I just change this to authSource, and PR this into master, everything will be fine when master is released? Changing `db` in `authSource` then :D